### ISSUE TITLE
Can't build - Source Directory "pipewire-screenaudio/native/pipewire-screenaudio/rohrkabel" does not contain a CMakeLists.txt file

### ISSUE THREAD

**Issue by TacoCake** [IceDBorn/pipewire-screenaudio#6]
I did the following commands: 
```
git clone https://github.com/IceDBorn/pipewire-screenaudio.git
cd pipewire-screenaudio/native
bash build.sh
```
However, the "rohrkabel" directory seems empty, which gives the following error when running `bash build.sh`.
```
-- The C compiler identification is GNU 13.1.1
-- The CXX compiler identification is GNU 13.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at pipewire-screenaudio/CMakeLists.txt:19 (add_subdirectory):
The source directory

/home/user/Downloads/pipewire-screenaudio/native/pipewire-screenaudio/rohrkabel

does not contain a CMakeLists.txt file.

-- Configuring incomplete, errors occurred!
```
I managed to narrow down the problem to the fact that it was a git submodule that wasn't initiated, which I fixed by running: `git submodule update --init --remote --recursive` at the root of the git repo.

A more permanent fix would be to include the `--recursive` argument when cloning this repo, you should update the README to include this.
In short, instead of:
`git clone https://github.com/IceDBorn/pipewire-screenaudio.git`
It should be:
`git clone --recursive https://github.com/IceDBorn/pipewire-screenaudio.git`

**Comment by IceDBorn** [comment]
You are right, I forgot to add the submodules flag, thanks a lot for clarifying it.
